browser/lib/markdown.js

```diff
@@ -149,6 +149,7 @@ class Markdown {
     })
     this.md.use(require('markdown-it-kbd'))
     this.md.use(require('markdown-it-admonition'))
+    this.md.use(require('markdown-it-front-matter'), fm => {})
```

Comment:
Why do you need an empty callback? Please remove it if you accidentally put it there.

Reply:
No accident, the plugin needs it or it won't work... It used so we can parse the front-matter to extract the properties. But since we don't need to do that, the code block is empty.

Reply:
Well that's annoying :smile: maybe because the plugin did check if the callback function is provided or not and assume that's there :smile: